bundle/internal/schema/main.go

```diff
@@ -21,6 +21,19 @@ func addInterpolationPatterns(typ reflect.Type, s jsonschema.Schema) jsonschema.
 		return s
 	}
 
+	// The variables block in a target override allows for directly specifying
+	// the value if it is a primitive type.
+	if typ == reflect.TypeOf(variable.TargetVariable{}) {
```

Comment:
Note: this is missing the schema for complex variables. I think that's a pattern that we should not encourage. Thoughts?

Reply:
That's OK, I expect this to be addressed in the big refactor change?

Reply:
Fixed in this PR itself based on our offline discussion. We'll allow all values including complex overrides.